### ISSUE TITLE
Multipass info does not work right after launching a instance

### ISSUE THREAD

**Issue by georgeliao** [canonical/multipass#3569]
**Describe the bug**
`multipass info` does not work right after launching an instance, it worked later without changing anything. 
**To Reproduce**
How, and what happened?
Just simply run `multipass launch -n vm4 bionic && multipass info vm4 -vvvv`

**Expected behavior**
`multipass info vm4 -vvvv` should work in regardless of how soon after launch finishes

**Logs**
```
Launched: vm4
[2024-07-09T15:33:00.576] [trace] [ssh session] Executing 'echo loadavg: "$(eval "cat /proc/loadavg | cut -d ' ' -f1-3")"; echo mem_usage: "$(eval "free -b | grep 'Mem:' | awk '{printf \$3}'")"; echo mem_total: "$(eval "free -b | grep 'Mem:' | awk '{printf \$2}'")"; echo disk_usage: "$(eval "df -t ext4 -t vfat --total -B1 --output=used | tail -n 1")"; echo disk_total: "$(eval "df -t ext4 -t vfat --total -B1 --output=size | tail -n 1")"; echo cpus: "$(eval "nproc")"; echo cpu_times: "$(eval "head -n1 /proc/stat")"; echo uptime: "$(eval "uptime -p | tail -c+4")"; echo current_release: "$(eval "cat /etc/os-release | grep 'PRETTY_NAME' | cut -d \\\" -f2")"'
[2024-07-09T15:33:00.594] [trace] [ssh process] /home/georgel/projects/canonical_projects/multipass/src/ssh/ssh_process.cpp:167 read_stream(type = 0, timeout = -1):
[2024-07-09T15:33:00.594] [trace] [ssh process] /home/georgel/projects/canonical_projects/multipass/src/ssh/ssh_process.cpp:189 read_stream(): num_bytes = 211
[2024-07-09T15:33:00.594] [trace] [ssh process] /home/georgel/projects/canonical_projects/multipass/src/ssh/ssh_process.cpp:189 read_stream(): num_bytes = 0
info failed: yaml-cpp: error at line 9, column 1: bad conversion
```

**Additional info**
 - OS: [Ubuntu 24.04]
- `multipass version`
- 1.15.0-dev.2327+gcbafad3fc and 1.14.0-rc5
- `multipass get local.driver`
- qemu

**Additional context**
The uptime field of the [line](https://github.com/canonical/multipass/blob/9a9bcf8305b577449d95293dad48d937f0ff240c/src/daemon/runtime_instance_info_helper.cpp#L97) is not set right after the VM is launched.

**Comment by Ethan47345** [comment]
![image](https://github.com/user-attachments/assets/a2195825-6b9d-4e73-a693-ff2c6b37591a)
Has it been resolved?

**Comment by Ethan47345** [comment]
> Hi @Ethan47345, what does `multipass version` say?

![image](https://github.com/user-attachments/assets/f6360e52-8118-407f-8b4b-6f68b5cabdd1)
I think it's the latest.

**Comment by Ethan47345** [comment]
![image](https://github.com/user-attachments/assets/004215fc-06a2-49e1-8131-543e3e0b05f8)
Is that the name correct?
 And no, it's always failed
![image](https://github.com/user-attachments/assets/29300237-7ef4-4dac-b4a3-922db779092c)
But I installed ubuntu-desktop in it before it broke down, I can connect it by using Remote Desktop(Even though I can't connect to it in GUI)
![image](https://github.com/user-attachments/assets/e66dbc34-0090-4649-b0c2-7e6b8b9a1129)

**Comment by georgeliao** [comment]
@Ethan47345 
I forgot that you are on windows so the `&&` does not work, and plus windows does not have bionic. Anyway, if `yaml-cpp: error` persists regardless of elapsed time. Then it smells like something else. I will try to reproduce the error. Did you launch docker blueprint via the simple command `multipass launch docker -n docker-dev`?

**Comment by Ethan47345** [comment]
@georgeliao 
yes, exactly the same as https://multipass.run/docs/tutorial#launch-from-a-blueprint-to-run-docker-containers.
Originally I just wanted to run docker, but then I thought it would be nice to just pair the development environment in this instance as well.....

**Comment by georgeliao** [comment]
@Ethan47345 
Unfortunately, I can not reproduce that on multipass 1.14 windows. Can you describe the detailed steps? Since this looks like a new issue, just file a new issue that includes the details based on the mutipass github issue template Thanks.

**Comment by Ethan47345** [comment]
@georgeliao 
To be honest, I have somewhat forgotten how I ended up like this. I can only say roughly when the problem started. Initially, I just followed the steps on the official website and there were no issues. Then I thought I might as well set up a complete development and deployment environment in this instance, so I installed ubuntu-desktop and xrdp. There were some Chinese character encoding problem in the browser, but at first, I thought it was a system issue (although I later found out it was actually a problem with the Firefox version). So I configured /etc/environment, and also installed compizconfig-settings-manager and unity. After installing them, I needed to reboot the system, and it seems that it was at this point I discovered I could no longer connect to this docker-dev instance via the GUI.
So even if I use a new device, I might not be able to reproduce the issue about **To Reproduce**. If I file a new issue, I can only summarize our previous conversation. Is that okay?
